app/modules/cron.c

```diff
@@ -104,7 +104,7 @@ static size_t lcron_findindex(lua_State *L, cronent_ud_t *ud) {
   size_t i;
   for (i = 0; i < cronent_count; i++) {
     lua_rawgeti(L, LUA_REGISTRYINDEX, cronent_list[i]);
-    eud = lua_touserdata(L, 1);
+    eud = lua_touserdata(L, -1);
```

Comment:
Not sure what it is... But you definitely know inner mechanics better.

Reply:
The rawgeti *pushes* the integer onto the stack. This is element -1. Element 1 is the first argument in the current lua stackframe.